models.py

```diff
@@ -78,7 +79,11 @@ class Identity(db.Model):
     __tablename__ = 'identity'
     identity = db.Column(db.String(42), primary_key=True)
     created_at = db.Column(db.DateTime)
+    signature = db.Column(db.Text)
```

Comment:
We dont need such data in DB.
If it's logging/debugging purposes, should go with table `api_log` - there all requests are logged

Reply:
models recovered to previous version, these loggings are out of scope